vocab/properties.json

```diff
@@ -54,13 +72,25 @@
             "SANDS/v3/atlas/parcellationEntity",
             "SANDS/v3/atlas/parcellationEntityVersion",
             "publications/v1/periodical"
-        ]
+        ],
+        "formating": {
+            "formattedText": false,
+            "multilineText": false
+        }
     },
     "https://openminds.ebrains.eu/vocab/about": {
         "description": null,
         "label": "About",
+        "labelForReverseLink": {
```

Comment:
This is not needed I think. The reverse link could actually be the same as the normal link ("about").